### DISCUSSION TITLE
Live QSO vs Post QSO

### DISCUSSION THREAD

**Discussion by soakland5** [magicbug/Cloudlog#3033] [General]
I have noticed that, when making log entries, it is easy to enter in a contact into POST QSO when you think it is LIVE QSO, resulting in the wrong UTC timer being entered. There is no indication which screen you are in. Isa this something that can be corrected? Perhaps a title just above that says "POST QSO" or "LIVE QSO" when in that screen? It has happened to me a lot, and then you havew to go back and correct the times on each entry before you realize you're in the wrong screen.  73 de Scott W1AL

**Comment by phl0** [comment]
Proposed a solution in https://github.com/magicbug/Cloudlog/pull/3034.
